frappe/printing/page/print_format_builder/print_format_builder.js

```diff
@@ -771,6 +771,12 @@ frappe.PrintFormatBuilder = Class.extend({
 						df.fieldtype = "HTML";
 						df.options = $($this.find(".html-content")[0]).data('content');
 					}
+					
+					if($this.attr("data-reptext")){
+						this.innerText = $this.attr("data-reptext");
+						alert($this.attr("data-reptext"));
```

Comment:
why alert?

Reply:
I'm sorry this was remaining from testing. I will remove it.